### PR TITLE
Update documentation for RockPi 4 booting

### PR DESCRIPTION
**Pull request by Leo-Yan**
This patch set is to update documentation for RockPi 4 booting.

The first patch gives details for how to use U-boot command to detect USB stick, this can help developer to know if the USB port is compatible with the inserted USB stick.

The second patch gives method for only using SD card.

### CODE REVIEW

**Comment by chaws** [comment]
LGTM, that's pretty much what I did to get my rockpi booting

**Review by jbech-linaro — CHANGES_REQUESTED** [review]

File: docs/install/rockpi4.rst [hunk 1]
```diff
@@ -83,17 +83,75 @@ To flash the rootfs image you built above, from your trs build directory
...
 .. code-block:: bash
 
 	$ efidebug boot add -b 1 BootLedge usb 0:1 efi/boot/bootaa64.efi -i usb 0:1 ledge-initramfs.rootfs.cpio.gz -s 'console=ttyS2,1500000 console=tty0 root=UUID=6091b3a4-ce08-3020-93a6-f755a22ef03b rootwait panic=60' ; efidebug boot order 1
 
+Only attach SD card
+===================
+
+Alternatively we can use only SD card to boot system and the USB stick is not
+required. In this case, we need to merge firmware and root file system images
+into a single image and burn it into the SD card, this can be finished with
```
[the remark this answers is not in the document]
Comment (Leo-Yan) [reply docs/install/rockpi4.rst]: Will fix.

File: docs/install/rockpi4.rst [hunk 2]
```diff
@@ -83,17 +83,75 @@ To flash the rootfs image you built above, from your trs build directory
...
+
+        $ gunzip ts-firmware-rockpi4b.wic.gz
+        $ wget https://git.linaro.org/ci/job/configs.git/plain/ledge/ts/scripts/ts-merge-images.sh
+        $ chmod +x ts-merge-images.sh
+        $ ./ts-merge-images.sh ts-firmware-rockpi4b.wic ewaol-baremetal-image-ledge-secure-qemuarm64.wic
+
+To boot from EFI partition in SD card, we need firstly get to know what's the
+partition number for the EFI system partition:
+
+.. code-block:: bash
```
[the remark this answers is not in the document]
Comment (Leo-Yan) [reply docs/install/rockpi4.rst:125]: Will fix.

**Comment by Leo-Yan** [comment]
Thanks for reviewing, @chaws and @jbech-linaro .  Have updated the patches for Joakim's comments.

**Pull request merged**